src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php

```diff
@@ -79,11 +82,11 @@ public function __construct(
                 ->setDefault('zone', LazyOption::randomOne($zoneRepository))
                 ->setAllowedTypes('zone', ['null', 'string', ZoneInterface::class])
                 ->setNormalizer('zone', LazyOption::findOneBy($zoneRepository, 'code'))
-                ->setDefault('shipping_category', LazyOption::randomOne($shippingCategoryRepository))
-                ->setAllowedTypes('shipping_category', ['null', 'string', ShippingCategoryInterface::class])
-                ->setNormalizer('shipping_category', LazyOption::findOneBy($shippingCategoryRepository, 'code'))
+                ->setDefined('category')
```

Comment:
It's something related to #5384, ShippingMethod has both `taxCategory` and `category` which can be confusing, that's why I decided to use `shipping_category` as name for this option.